cmd/crc-embedder/cmd/embed.go

```diff
@@ -52,9 +54,10 @@ func runEmbed(args []string) {
 	if err != nil {
 		logging.Fatalf("Failed to download data files: %v", err)
 	}
-
-	bundlePath := path.Join(bundleDir, constants.GetDefaultBundleForOs(goos))
-	downloadedFiles = append(downloadedFiles, bundlePath)
+	if !excludeBundle {
+		bundlePath := path.Join(bundleDir, constants.GetDefaultBundleForOs(goos))
+		downloadedFiles = append(downloadedFiles, bundlePath)
+	}
```

Comment:
`crc` has subtle behaviour differences for releases/not releases, and this is triggered by the bundle being embedded or not https://github.com/code-ready/crc/blob/dccffe6e6981203ac5cb0b96cf54afafb9beb63d/pkg/crc/constants/constants.go#L153-L156
So this would need careful testing :)

Reply:
Actually, looking again at that code, it seems with the macOS installer, the bundle is *not* embedded? Might make sense to be consistent on Windows, and not embed it as well? This probably can be changed later though  without too much trouble, so thist can be postponed?

Reply:
Yeah, plan to do this in a followup PR, lets test the signing works for us with this initial msi